workspaces/azureml_devtestlabs/readme.md

```diff
@@ -0,0 +1,46 @@
+# Azure ML Worksapce
```

Comment:
I would be great with a detailed description and a diagram showing what is provisioned

Reply:
Agree, will look at that once get all the components in. Still need VMs etc to be able to access Azure ML.